cirq-ionq/cirq_ionq/job_test.py

```diff
@@ -254,7 +254,7 @@ def test_job_results_poll(mock_sleep):
     mock_client.get_results.return_value = {'0': '0.6', '1': '0.4'}
     job = ionq.Job(mock_client, ready_job)
     results = job.results(polling_seconds=0)
-    assert results[0] == ionq.QPUResult({0: 600, 1: 400}, 1, measurement_dict={})
+    assert results == ionq.QPUResult({0: 600, 1: 400}, 1, measurement_dict={})
```

Comment:
need multi-results tests now, I think

Reply:
we actually have those as well, this just reverts the single-results tests to not index lists